qiskit_nature/second_q/problems/base_problem.py

```diff
@@ -77,6 +77,15 @@ def second_q_ops(self) -> tuple[SparseLabelOp, dict[str, SparseLabelOp]]:
 
         return main_op, aux_ops
 
+    @deprecate_function(
+        "0.6.0",
+        additional_msg=(
+            ". This function is deprecated because it has been removed from the public API. It is "
```

Comment:
> This function is deprecated because it has been removed from the public API.

We can't have already removed it right - its deprecated as it will be removed.

Reply:
Yeh I phrased that poorly.. It is not actually removed yet but will be in the future :+1: